core/src/main/java/nl/nn/adapterframework/http/HttpSender.java

```diff
@@ -546,36 +551,30 @@ protected Message extractResult(HttpResponseHandler responseHandler, IPipeLineSe
 				} catch (IOException e) {
 					throw new SenderException("cannot find filename to stream result to", e);
 				}
-			} else if (isBase64()) {
+			} else if (isBase64()) { //This should be removed in a future iteration
 				return getResponseBodyAsBase64(responseHandler.getResponse());
-			} else if (StringUtils.isNotEmpty(getStoreResultAsStreamInSessionKey())) {
-				session.put(getStoreResultAsStreamInSessionKey(), responseHandler.getResponse());
-				return Message.nullMessage();
-			} else if (StringUtils.isNotEmpty(getStoreResultAsByteArrayInSessionKey())) {
-				session.put(getStoreResultAsByteArrayInSessionKey(), Misc.streamToBytes(responseHandler.getResponse()));
+			} else if (StringUtils.isNotEmpty(storeResultInSessionKey)) { //This should be removed in a future iteration
+				Message message = new Message(responseHandler.getResponse());
+				session.put(storeResultInSessionKey, message);
```

Comment:
Je stopt nu een Message in de sessie, in plaats van een Stream of byte array. Het is maar de vraag of de ontvangende partijen daarmee om kunnen gaan.

Reply:
Gelukkig is dat wel het geval. Deze wordt namelijk als parameter meegegeven, en die kan met Message objecten overweg. Zal hier nog een extra test voor maken.